ajax/libs/deep-freeze/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "deep-freeze",
+  "version": "0.0.1",
+  "description": "recursively Object.freeze() objects and functions",
+  "filename": "index.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/substack/deep-freeze.git"
+  },
+  "keywords": [
+    "freeze",
+    "deep",
+    "object",
+    "recursive"
+  ],
+  "author": {
+    "name": "James Halliday",
+    "email": "mail@substack.net",
+    "url": "http://substack.net"
+  },
+  "license": "public domain",
```

Comment:
https://developer.mozilla.org/en-US/docs/MDN/About#Copyrights_and_licenses
you can find SPDX License in https://spdx.org/licenses/